src/components/reusable/errorBlock/errorBlock.stories.js

```diff
@@ -0,0 +1,74 @@
+import { html } from 'lit';
+import './index';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+
+import '@kyndryl-design-system/shidoka-foundation/components/button';
+import alertImg from './redfords/Alert_Redford.svg';
+import notFoundImg from './redfords/NotFound_Redford.svg';
+import stopImg from './redfords/Stop_Redford.svg';
+import timeoutImg from './redfords/Timeout_Redford.svg';
+
+export default {
+  title: 'Components/Error Block',
+  component: 'kyn-error',
+  parameters: {
+    design: {
+      type: 'figma',
+      url: 'https://www.figma.com/design/CQuDZEeLiuGiALvCWjAKlu/Applications---Component-Library?node-id=17386-106044&node-type=canvas&t=VYrUi3e41lGwsAHM-0',
+    },
+  },
+};
+
+const args = {
+  titleText: 'Error title',
+};
+
+export const ErrorBlock = {
+  args,
+  decorators: [
+    (story) => html`
+      <style>
+        /* Example styles. Added to make the action buttons responsive. */
+        .actions-container {
+          display: flex;
+          gap: 16px;
+          flex-wrap: wrap;
```

Comment:
These styles should be internal to the component, on the `error-action-buttons` div.

Reply:
Tried the same, but the css doesn't seem to be applying when it is put on the component.

Reply:
Slotted elements should be affected by flex properties on the parent container.

Reply:
Yes, it's working, made a mistake in the slot name, thanks!